pagopa/io-app
# feat: [PE-549] Adopt DS on CGN detail screen

## Short description
This PR refactors the CGN detail screen to adopt the new DS Components

## List of changes proposed in this pull request
- Changes `CgnDetailScreen`
- Changes every child component for status information rendering

## How to test
Check the CGN detail screen to be properly rendered.

https://github.com/pagopa/io-app/assets/3959405/ccdfff2c-f789-4490-b2a3-00a326af54cc 

| CGN Status | Screen |
|--------|--------|
| Active | Check below video 👇 |
| Revoked | <img src="https://github.com/pagopa/io-app/assets/3959405/33a64029-e6da-4944-a647-13a0ec37a6c0" height="500" /> |
| Expired | <img src="https://github.com/pagopa/io-app/assets/3959405/044ac6da-af28-4b6d-883b-8d74f47b9e17" height="500" /> |

| Eyca Status | Screen |
|--------|--------|
| Active | <img src="https://github.com/pagopa/io-app/assets/3959405/3bc2dc99-ea1e-4679-abd0-f07047b4bded" height="500" /> |
| Pending | <img src="https://github.com/pagopa/io-app/assets/3959405/5e759079-e4c3-4dd5-86c1-52c45147a343" height="500" /> |
| Error | <img src="https://github.com/pagopa/io-app/assets/3959405/b02b3a33-f358-48ab-9358-7633d8e66529" height="500" /> |
